core/src/main/resources/jenkins/model/Messages_de.properties

```
@@ -21,8 +21,8 @@
 # THE SOFTWARE.
 
 Hudson.BadPortNumber=Falsche Portnummmer {0}
-Hudson.Computer.Caption=Master
-Hudson.Computer.DisplayName=master
+Hudson.Computer.Caption=Eingebauter Knoten
+Hudson.Computer.DisplayName=Eingebaut
```

```suggestion
Hudson.Computer.Caption=Interner Knoten
Hudson.Computer.DisplayName=Intern
```
Applying Daniel's suggestions